src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -72,6 +73,11 @@ public function __construct(
         array $srcDirs,
         bool $skipCoverage
     ) {
+        Assert::notEmpty(
```

Comment:
Add a guard against this case since otherwise we get a cryptic error instead of more "graceful" one L96